force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1849,6 +1849,30 @@
         <shortDescription>Text indicating the Recurring Donation could not be created in Elevate</shortDescription>
         <value>An Elevate recurring payment could not be created. Contact your system administrator for assistance.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevateRDCancellingTitle</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text indicating the Recurring Donation would be cancelled</shortDescription>
+        <value>Your request to stop your Recurring Donation was submitted.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_ElevateRDCancellingMessage</fullName>
+        <categories>Recurring Donation</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text indicating the Recurring Donation would be cancelled</shortDescription>
+        <value>This can take some time. Refresh the page to check the status. Contact us if you need help.</value>
```

Comment:
Are these Labels all approved? "Contact us if you need help." doesn't seem like something we'd normally put in a message 🤷

Reply:
@lparrott the labels are defined in this document https://salesforce.quip.com/kRYaAjEE8aHr the text is as described.